src/scenes/AcceleratorRouter/People/RemoveRolesTopBarButton.tsx

```diff
@@ -0,0 +1,61 @@
+import React, { useCallback, useState } from 'react';
+
+import { Typography } from '@mui/material';
+import { Button, DialogBox } from '@terraware/web-components';
+
+import strings from 'src/strings';
+
+interface RemoveRolesTopBarButtonProps {
+  onConfirm: () => void;
+}
+
+function RemoveRolesTopBarButton({ onConfirm }: RemoveRolesTopBarButtonProps) {
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false);
+
+  const _onConfirm = useCallback(() => {
+    onConfirm();
+    setIsModalOpen(false);
+  }, [onConfirm]);
+
+  return (
+    <>
+      <Button
+        icon='iconTrashCan'
+        onClick={() => setIsModalOpen(true)}
+        label={strings.REMOVE_ACCESS}
+        priority={'secondary'}
+        type={'destructive'}
+      />
+
+      <DialogBox
+        onClose={() => setIsModalOpen(false)}
+        open={isModalOpen}
+        title={strings.REMOVE_ACCESS}
+        size={'medium'}
+        middleButtons={[
+          <Button
+            label={strings.CANCEL}
+            priority='secondary'
+            type='passive'
+            onClick={() => setIsModalOpen(false)}
+            key='button-1'
+            id='cancelButton'
+          />,
+          <Button
+            label={strings.REMOVE_ACCESS}
```

Comment:
Just a heads up, I have a ticket open https://terraformation.atlassian.net/browse/SW-4969 to componentize confirmation dialogs since we have a lot of boilerplate. When we get to that, we could make the callback async and await on it.. disable the confirm button while the callback is executing.